front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -137,6 +137,13 @@ const AuthProviders = memo<Props & WrappedComponentProps>(
     const phone =
       !isNilOrError(tenant) && tenant.attributes.settings.password_login?.phone;
 
+    const renderPasswordAuthButton =
+      passwordLoginEnabled &&
+      (flow === 'signin' ||
```

Comment:
Just to make sure I understand:

This is true if `passwordLoginEnabled` and flow is signin, or true if `passwordLoginEnabled`, flow is `signup` and the new `enable_signup` attribute is also true

Reply:
Yes, it's a bit confusing because the existing feature flag has that name already. The behaviour is like this:


|  `password_login` |  `enable_signup` | Login possible | Signup possible |
|-------------------|------------------|----------------|-----------------|
| false             | false            | false          | false           |
| true              | false            | true           | false           |
| true              | true             | true           | true            |


If I can make this a bit clearer in the code, let me know. The additional conditional check here is the `flow`. We need to check if the current action is a `signin` or `signup`.